BackupDr/samples/V1/BackupDRClient/list_management_servers.php

```diff
@@ -33,11 +33,10 @@
  * Lists ManagementServers in a given project and location.
  *
  * @param string $formattedParent The project and location for which to retrieve management servers
- *                                information, in the format 'projects/{project_id}/locations/{location}'. In
- *                                Cloud BackupDR, locations map to Google Cloud regions, for example
- *                                **us-central1**. To retrieve management servers for all locations, use "-"
- *                                for the
- *                                '{location}' value. Please see
+ *                                information, in the format `projects/{project_id}/locations/{location}`. In
+ *                                Cloud BackupDR, locations map to GCP regions, for example **us-central1**.
+ *                                To retrieve management servers for all locations, use "-" for the
+ *                                `{location}` value. Please see
  *                                {@see BackupDRClient::locationName()} for help formatting this field.
```

Comment:
Is the nested @see correct here? I would've expected to have the link here 🤔

Reply:
It looks correct to me - the `{@see [REF]}` syntax can be placed inside parameter descriptions as well.